internal/cli/apps.go

```diff
@@ -90,12 +92,12 @@ var CmdAppCreate = &cobra.Command{
 			return nil, cobra.ShellCompDirectiveNoFileComp
 		}
 
-		app, err := clients.NewEpinioClient(cmd.Context())
+		app, err := clients.NewEpinioClient(context.Background())
```

Comment:
Some of these auto-completion functions take quite long to complete. Maybe we could create one context with Timeout (https://pkg.go.dev/context#WithTimeout) at the beginning and use it in all commands.

Reply:
I would consider that to be out of scope for this bug fix. IMHO better done as a separate ticket.